tests/test_notifications.py

```
@@ -151,3 +151,87 @@ async def test_generic_exception_wraps(self, _mock_graphql: AsyncMock) -> None:
         tool_fn = _make_tool()
         with pytest.raises(ToolError, match="boom"):
             await tool_fn(action="overview")
+
+
+class TestNotificationsCreateValidation:
+    """Tests for importance enum and field length validation added in this PR."""
+
+    async def test_invalid_importance_rejected(self, _mock_graphql: AsyncMock) -> None:
```

Ruff (0.15.1)

[warning] Fixture `_mock_graphql` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)